### PR TITLE
DDP-7292 Add showActivityStatus to activity definition

### PR DESCRIPTION
**Pull request by ssettipalli**
look at https://broadinstitute.atlassian.net/browse/DDP-7292
New column will be used by DSM

Once we get confirmation if we want to set the new flag for existing studies, we might need to back fill/update the new column as needed.
Might need to tweak little bit later once we configure FON depending on DSM need

### CODE REVIEW

File: pepper-apis/src/main/java/org/broadinstitute/ddp/model/activity/definition/ActivityDef.java [hunk 1]
```diff
@@ -357,6 +360,10 @@ public Boolean getCanDeleteFirstInstance() {
         return canDeleteFirstInstance;
     }
 
+    public boolean isShowActivityStatus() {
```
Comment (bskinner) [inline pepper-apis/src/main/java/org/broadinstitute/ddp/model/activity/definition/ActivityDef.java]: This reads strangely- `showActivityStatus` is already a true/false statement, prefixing the name with `is` isn't necessary

**Review by bskinner — DISMISSED** [review]
Overall, this looks good! I did leave a comment about the naming of the getter for `showActivityStatus`.

File: pepper-apis/src/main/java/org/broadinstitute/ddp/model/activity/definition/ActivityDef.java (hunk 1 above)
Comment (ssettipalli) [reply pepper-apis/src/main/java/org/broadinstitute/ddp/model/activity/definition/ActivityDef.java]: renamed

**Review by bskinner — APPROVED** [review]
👍

**Pull request merged**